python/tox.ini

```diff
@@ -25,6 +25,10 @@ deps =
     mock
     pytest
     pyarrow
+    pytz
```

Comment:
Should we define those here as well? I would argue that we already set these in `setup.cfg`, and if we start pinning them to certain versions, it is better to set this in one place.